spec.html

```diff
@@ -358,10 +358,15 @@ <h3>Notes</h3>
       type uses delta and delta-of-delta compression and
```

Comment:
Can we rework this section into a table and give it a more appropriate heading than "Notes"? I'd suggest something but I'm not quite sure what this is about.